<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSongDialog</class>
 <widget class="QDialog" name="EditSongDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Song Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/openlp.org-icon-32.bmp</normaloff>:/icon/openlp.org-icon-32.bmp</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QTabWidget" name="SongTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="LyricsTab">
      <attribute name="title">
       <string>Title &amp;&amp; Lyrics</string>
      </attribute>
      <layout class="QGridLayout" name="LyricsTabLayout">
       <property name="margin">
        <number>8</number>
       </property>
       <property name="spacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="TitleLabel">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="TitleEditItem">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="AlternativeTitleLabel">
         <property name="text">
          <string>Alternative Title:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="AlternativeEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="LyricsLabel">
         <property name="text">
          <string>Lyrics:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QListWidget" name="VerseListWidget"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="VerseOrderLabel">
         <property name="text">
          <string>Verse Order:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="VerseOrderEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QWidget" name="VerseButtonWidget" native="true">
         <layout class="QVBoxLayout" name="VerseButtonsLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="VerseAddButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="VerseEditButton">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="VerseEditAllButton">
            <property name="text">
             <string>Edit All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="VerseDeleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="VerseSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AuthorsTab">
      <attribute name="title">
       <string>Authors, Topics &amp;&amp; Book</string>
      </attribute>
      <layout class="QHBoxLayout" name="AuthorsTabLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item>
        <widget class="QWidget" name="AuthorsMaintenanceWidget" native="true">
         <layout class="QVBoxLayout" name="AuthorsMaintenanceLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="AuthorsGroupBox">
            <property name="title">
             <string>Authors</string>
            </property>
            <layout class="QVBoxLayout" name="AuthorsLayout">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="margin">
              <number>8</number>
             </property>
             <item>
              <widget class="QWidget" name="AuthorAddWidget" native="true">
               <layout class="QHBoxLayout" name="AuthorAddLayout">
                <property name="spacing">
                 <number>8</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="AuthorsSelectionComboItem">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="editable">
                   <bool>false</bool>
                  </property>
                  <property name="insertPolicy">
                   <enum>QComboBox::InsertAlphabetically</enum>
                  </property>
                  <property name="sizeAdjustPolicy">
                   <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                  </property>
                  <property name="minimumContentsLength">
                   <number>8</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="AuthorAddButton">
                  <property name="maximumSize">
                   <size>
                    <width>110</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&amp;Add to Song</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="AuthorsListView">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="AuthorRemoveWidget" native="true">
               <layout class="QHBoxLayout" name="AuthorRemoveLayout">
                <property name="spacing">
                 <number>8</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <spacer name="AuthorsSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="AuthorRemoveButton">
                  <property name="text">
                   <string>&amp;Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="MaintenanceWidget" native="true">
            <layout class="QHBoxLayout" name="MaintenanceLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="MaintenanceButton">
               <property name="text">
                <string>&amp;Manage Authors, Topics, Books</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="MaintenanceSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>66</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="TopicBookWidget" native="true">
         <layout class="QVBoxLayout" name="TopicBookLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="TopicGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Topic</string>
            </property>
            <layout class="QVBoxLayout" name="TopicLayout">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="margin">
              <number>8</number>
             </property>
             <item>
              <widget class="QWidget" name="TopicAddWidget" native="true">
               <layout class="QHBoxLayout" name="TopicAddLayout">
                <property name="spacing">
                 <number>8</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="SongTopicCombo">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="TopicAddButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>A&amp;dd to Song</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="TopicsListView">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="TopicRemoveWidget" native="true">
               <layout class="QHBoxLayout" name="TopicRemoveLayout">
                <property name="spacing">
                 <number>8</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <spacer name="TopicsSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="TopicRemoveButton">
                  <property name="text">
                   <string>R&amp;emove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="SongBookGroup">
            <property name="title">
             <string>Song Book</string>
            </property>
            <layout class="QGridLayout" name="SongbookLayout">
             <property name="margin">
              <number>8</number>
             </property>
             <property name="spacing">
              <number>8</number>
             </property>
             <item row="0" column="0">
              <widget class="QComboBox" name="SongbookCombo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ThemeTab">
      <attribute name="title">
       <string>Theme, Copyright Info &amp;&amp; Comments</string>
      </attribute>
      <layout class="QVBoxLayout" name="ThemeTabLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="margin">
        <number>8</number>
       </property>
       <item>
        <widget class="QWidget" name="ThemeCopyCommentsWidget" native="true">
         <layout class="QHBoxLayout" name="ThemeCopyCommentsLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="TextWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="DetailsLayout">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="ThemeGroupBox">
               <property name="title">
                <string>Theme</string>
               </property>
               <layout class="QHBoxLayout" name="ThemeLayout">
                <property name="spacing">
                 <number>8</number>
                </property>
                <property name="margin">
                 <number>8</number>
                </property>
                <item>
                 <widget class="QComboBox" name="ThemeSelectionComboItem"/>
                </item>
                <item>
                 <widget class="QPushButton" name="ThemeAddButton">
                  <property name="maximumSize">
                   <size>
                    <width>110</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Add a Theme</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="CopyrightGroupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Copyright Information</string>
               </property>
               <layout class="QVBoxLayout" name="CopyrightLayout">
                <property name="spacing">
                 <number>8</number>
                </property>
                <property name="margin">
                 <number>8</number>
                </property>
                <item>
                 <widget class="QWidget" name="CopyrightWidget" native="true">
                  <layout class="QHBoxLayout" name="CopyLayout">
                   <property name="spacing">
                    <number>8</number>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="CopyrightEditItem"/>
                   </item>
                   <item>
                    <widget class="QPushButton" name="CopyrightInsertButton">
                     <property name="maximumSize">
                      <size>
                       <width>29</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>©</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="CcliWidget" native="true">
                  <layout class="QHBoxLayout" name="CCLILayout">
                   <property name="spacing">
                    <number>8</number>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="CCLILabel">
                     <property name="text">
                      <string>CCLI Number:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="CCLNumberEdit"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="ThemeCopySpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="CommentsGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Comments</string>
            </property>
            <layout class="QVBoxLayout" name="CommentsLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>8</number>
             </property>
             <item>
              <widget class="QTextEdit" name="CommentsEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="ThemeTabSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SongTabWidget</tabstop>
  <tabstop>TitleEditItem</tabstop>
  <tabstop>AlternativeEdit</tabstop>
  <tabstop>VerseListWidget</tabstop>
  <tabstop>VerseAddButton</tabstop>
  <tabstop>VerseEditButton</tabstop>
  <tabstop>VerseEditAllButton</tabstop>
  <tabstop>VerseDeleteButton</tabstop>
  <tabstop>VerseOrderEdit</tabstop>
  <tabstop>AuthorsSelectionComboItem</tabstop>
  <tabstop>AuthorAddButton</tabstop>
  <tabstop>AuthorsListView</tabstop>
  <tabstop>AuthorRemoveButton</tabstop>
  <tabstop>MaintenanceButton</tabstop>
  <tabstop>SongTopicCombo</tabstop>
  <tabstop>TopicAddButton</tabstop>
  <tabstop>TopicsListView</tabstop>
  <tabstop>TopicRemoveButton</tabstop>
  <tabstop>SongbookCombo</tabstop>
  <tabstop>ThemeSelectionComboItem</tabstop>
  <tabstop>ThemeAddButton</tabstop>
  <tabstop>CopyrightEditItem</tabstop>
  <tabstop>CopyrightInsertButton</tabstop>
  <tabstop>CCLNumberEdit</tabstop>
  <tabstop>CommentsEdit</tabstop>
  <tabstop>ButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../images/openlp-2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSongDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSongDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>-50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
